Adyen/Core/Core Protocols/DropInComponentProtocol.swift

```diff
@@ -0,0 +1,22 @@
+//
+// Copyright (c) 2022 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Foundation
+
+/// Any DropIn Component.
+public protocol DropInComponentProtocol: PresentableComponent {
```

Comment:
Nit pick but how about `AnyDropInComponent`? Would match the description too :D

Reply:
sure I can rename it to `AnyDropInComponent, hahaha